src/lib/host/taskcluster-worker-runner.js

```diff
@@ -37,6 +37,10 @@ module.exports = {
       throw new Error('No config file found');
     }
     const content = fs.readFileSync(configFile, 'utf8');
-    return JSON.parse(content);
+    return Object.assign({
+      capacityManagement: {
+        diskspaceThreshold: 30000000000,
```

Comment:
The default for this is already set (but to a different value) at
https://github.com/taskcluster/docker-worker/blob/4cad67ba1759c5676a5c52ad115725a7fb6f5067/config.yml#L67-L68
Wouldn't it make more sense to update that value, rather than default it here?